src/app/services/authorization/authorization.service.ts

```diff
@@ -1,29 +1,76 @@
 import { Injectable } from '@angular/core';
 import { UserModel } from '../../core/user.model';
 import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { Observable, ReplaySubject } from 'rxjs';
+import { LoadingService } from '../loading/loading.service';
+
+const LOGIN_PATH = 'http://localhost:3004/auth/login';
```

Comment:
Also you may create something like `ApiInterceptor` and change you urls to divide endpoints and host.